lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -329,6 +329,7 @@
     <datatype extension='qiime2.tabular' type="galaxy.datatypes.qiime2:QIIME2Metadata" display_in_upload="true"/>
     <datatype extension="zip" type="galaxy.datatypes.binary:CompressedZipArchive" display_in_upload="true">
         <converter file="archive_to_directory.xml" target_datatype="directory" />
+        <converter file="archive_to_directory.xml" target_datatype="lexicmap_index"/>
```

Comment:
This doesn't really make sense to me, that means every zip file is also a lexicmap_index.